data/human/human missions.txt

```diff
@@ -2514,6 +2514,139 @@ event "terraforming follow-up"
 
 
 
+mission "Terraforming 13"
+	minor
+	name "Field Trip"
+	description "Amy, who you helped terraform Rand and Tundra, seems to share some of her terraforming philosophy with the Wanderers. Bring her to <destination> to visit them."
+	source "Tundra"
+	destination "Desi Seledrak"
+	passengers 1
+	to offer
+		has "event: terraforming follow-up"
+		has "wanderers sestor done"
+	on offer
+		conversation
+			branch repeat
+				has "terraforming field trip: deferred later"
+			`As you walk through the spaceport, you notice a poster featuring Amy, whom you previously helped terraform this world and Rand. The poster describes that terraforming action, and the long-term goal to restore the warmer climate Tundra had millions of years ago. Amy's work on "equilibrium mapping," the idea that planets naturally "want" to support more complex and varied life, seems similar to the Wanderer philosophy that each planet has its own "will", and that the best way to terraform it is to work with that "will." Amy may be interested in learning about the Wanderers and their techniques. Would you like to find her and tell her about them?`
+			choice
+				`	(Yes.)`
+				`	(Not right now.)`
+					defer
+				`	(No.)`
+					decline
+			`	The poster includes small writing with contact instructions for inquiries about the terraforming process and Tundra's climate. Once you get through to a person on the other side, there is initially reluctance to put you through to anyone involved with the terraforming directly, but after verifying that you worked with them to get the process started, you are connected to Amy.`
+			`	You briefly summarize your adventures, including about the Wanderers: "Aliens with an approach to terraforming that matches your theories from when you were at the Academy of Planetary Sciences." She seems interested in learning more and agrees to meet you in the spaceport bar.`
```

Comment:
Including what about the Wanderers? Maybe remove the word "about"?

Reply:
The text that follows is what about the Wanderers we tell her.